mautic/mautic
# Email stats showing incorrect data in A/B testing mode

| Q  | A
| --- | ---
| Bug fix? | Y
| Issues addressed (#s or URLs) | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Email view page offers some statistics about activities. When working with A/B testing mode, you have posibility to view either statics for the corresponding e-mail or all email variants. The feature is not working as expected. It shows the same statistics for current e-mail variant and all variant (link available in the upper right corner of Email Stats chart). This PR fixes that issue and will show data as expected.

#### Steps to test this PR:
1. Create New Template Email
2. Add A/B test for that email, this will create new variant for that email
3. Use that email in campaign
4. When viewing that email, it will show statistics of current email variant and all email variants.
5. Check the email stats chart , it will show different data when switching between Variant and All.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create New Template Email
2. Add A/B test for that email, this will create new variant for that email
3. Use that email in campaign
4. When viewing that email, it will show statistics of current email variant and all email variants.
5. Check the email stats chart , it will show same data when switching between Variant and All.
